front/text/plain/convert_test.go

```diff
@@ -145,7 +145,7 @@ func TestToHTML_Plain(t *testing.T) {
 
 func TestToHTML_LineBreak(t *testing.T) {
 	post := "this is a line\nthis is another line"
-	expected := `<p>this is a line</p><p>this is another line</p>`
+	expected := `<p>this is a line<br/>this is another line</p>`
```

Comment:
Bug:

`"\n\n\nthis is a line\nthis is another line"`

becomes

`"<p></p><p><br/>this is a line<br/>this is another line</p>"`

Reply:
What would you want it to be?

Reply:
<p></br></br></br>this is a line</br>this is another line<p>

If you think that'll look neater, I'll make it so 💁🏻‍♀️

Reply:
`"<br/><br/><br/><p>this is a line<br/>this is another line</p>"` makes the most sense, no?

Reply:
> `"<br/><br/><br/><p>this is a line<br/>this is another line</p>"` makes the most sense, no?

Unfortunately not since <br/> isn't a block level element. It needs to be "inside" block level elements like p or h1.

Reply:
Oh no

Reply:
Pretty happy with the semantics of the most recent commit. It also wraps the whole thing in <p> no matter how many lines there are, again to fit in with the block level aspect of the HTML spec. And that cuts out the string splitting that you wanted to cut, too 👍🏻

Reply:
Small thing: move the MustCompile() call to an init() function, so it's called only once and panics before main() if it fails